soh/macosx/soh-macos.sh

```diff
@@ -1,8 +1,69 @@
-#!/bin/sh
-BUNDLE="`echo "$0" | sed -e 's/\/Contents\/MacOS\/.*//'`"
-RESOURCES="$BUNDLE/Contents/Resources"
+#!/usr/bin/env bash
```

Comment:
does this not work with `/bin/bash` all macs (at least currently) have bash3.2, using /usr/bin/env bash might load a newer one on your system, if you've installed it.